packages/shared/lib/core/wallet/interfaces/mint-nft-details.interface.ts

```diff
@@ -0,0 +1,17 @@
+import { TokenStandard } from '../enums'
+import { MimeType } from '../types'
+
+export interface IMintNftDetails {
```

Comment:
we have the `INftMetadata` interface with exactly the same fields, so lets use that